pkg/webhooks/router/admission.go

```diff
@@ -49,7 +49,7 @@ func RegisterAdmission(service *AdmissionService) error {
 }
 
 func ForEachAdmission(config *options.Config, handler func(*AdmissionService)) {
-	admissions := strings.Split(config.EnabledAdmission, ";")
+	admissions := strings.Split(config.EnabledAdmission, ",")
```

Comment:
one more suggestion: it's better to use `strings.TrimSpace` remove extra spaces first.

Reply:
done